app/models/review.rb

```diff
@@ -0,0 +1,8 @@
+class Review < ApplicationRecord
+  belongs_to :user
+  belongs_to :restaurant
+
+  validates :title, presence: true, length: { in: 3..30 }
+  validates :content, presence: true, length: { in: 5..300 }
```

Comment:
i feel like title and content should be optional to encourage users to review?

Reply:
Let's remove content and title then